charts/seed-monitoring/charts/prometheus/values.yaml

```diff
@@ -189,6 +189,16 @@ allowedMetrics:
   vpn:
   - probe_http_status_code
   - probe_success
+  elasticsearch:
+  - elasticsearch_cluster_health_json_parse_failures
+  - elasticsearch_cluster_health_total_scrapes
+  - elasticsearch_cluster_health_up
+  - elasticsearch_index_stats_json_parse_failures
+  - elasticsearch_index_stats_total_scrapes
+  - elasticsearch_index_stats_up
+  - elasticsearch_node_stats_json_parse_failures
```

Comment:
`*_json_parse_failures`, `*_total_scrapes` does not bring much value. You could remove them and more elastic critical ones.

Reply:
I changed the metrics. Can you check them now?